### ISSUE TITLE
Can I use Ax for models being trained on multiple gpus?

### ISSUE THREAD

**Issue by AmmaraRazzaq** [facebook/Ax#718]
Can Ax be used to do hyperparameter tuning for a model that is being trained on multiple gpus using DistributedDataParallel in Pytorch?

**Comment by danielcohenlive** [comment]
Thanks for the question @AmmaraRazzaq. Yes, you just need to provide it a metric or evaluation function, depending on which API you're using, so that it can perform optimization.  Where the model is being trained is not important to Ax, as it treats the thing it's optimizing as a blackbox.

**Comment by danielcohenlive** [comment]
Just in case I'm missing something, could you elaborate on what the challenges would be on either the parameter or the evaluation side of things?

**Comment by lena-kashtelyan** [comment]
Hi @AmmaraRazzaq, closing this issue as inactive and assuming your question was answered. Please reopen this issue and follow up if it was not!
